### DISCUSSION TITLE
Issues building Kade Engine 1.7 on Mac

### DISCUSSION THREAD

**Discussion by Incendispike** [KadeArchive/Kade-Engine#2275] [Q&A]
So I'm trying to build Kade Engine on Mac for a mod I have planned, but after installing all the dependencies and running 'lime test mac,' it goes through a bunch of warnings related to Lua and shoots back a 'linker command failed with exit code 1' error.

Is there something I'm missing, do I have to use a previous version, or does this just not work on Mac?

**Comment by isakube** [comment]
just use 1.5.4. dont even bother with 1.6. The charter sucks.

**Reply by Incendispike** [reply]
Well the game now builds 'properly' with 1.5.4, however it always crashes when trying to load everything. I haven't added anything new yet, it's purely the 1.5.4 download.
